R/cell_suppression.R

```diff
@@ -437,7 +431,9 @@ render_median.continuous <- function(x, ...) {
 #' A continuous variable to summarize.
 #'
 #' @param ... \cr
-#' Further arguments, passed to `table1:::stats.apply.rounding()`.
+#' Further arguments, such as `continuous_fn`, or those passed to `table1:::stats.apply.rounding()`.
+#' Use `continuous_fn` to specify the summary statistics to display, 
+#' which accepts character string: "mean", "median", or c("mean", "median") to display both. Defaults to "mean".
```

Comment:
Thanks for reviewing @loffleraSMH! The `render_cell_suppression.continuous` renderer can only accept following values: `mean`, `median` or `c("mean", "median")`, not `c("Mean (SD)", "Median [Q1, Q3]")`. The example in documentation shows a more "complicated" case to display both mean and median at the same time with `c("mean", "median")` - is this the example you are looking for?

Reply:
Ah yes, you are totally right! Sorry, I got confused because I initially assumed the inputs for this work the same way as if you specify `render.continuous`, where you need to explicitly specify if you want to show the measures of spread. Realizing now that SD/Q1,Q3 is shown by default when specifying `continuous_fn` the way you set it up - that makes a lot of sense. Ignore my comment then - and sorry for the confusion!

Reply:
No prob at all! Thanks again for reviewing I will go ahead and merge this! :)